src/inverse-kinematics/src/ConvexHullHelpers.cpp

```diff
@@ -342,5 +342,33 @@ namespace iDynTree
         return margin;
     }
 
+    void ConvexHullProjectionConstraint::setProjectionAlongDirection(Vector3 direction, const iDynTree::Position originOfPlaneInWorld)
+    {
+        o = originOfPlaneInWorld;
+
+        Vector3 xProjection, yProjection;
+
+        // define the projection for the x-component
+        xProjection.setVal(0, 1.0);
```

Comment:
I just realized that in this expression we are assuming that `xAxisPlaneInWorld` is equal to `(1, 0, 0)` and `yAxisPlaneInWorld` is `(0, 1, 0)`. For the time being it is ok to keep like this until we fully debug all the problems, but we should at least document it in the method documentation.